runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,356 @@
+			assert_ne!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::WeightInfo>(
+				),
+				TypeId::of::<()>(),
+			);
+		}
+	}
+
+	mod membership {
+		use super::*;
+
+		#[test]
+		fn add_origin_requires_unanimous_vote() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::AddOrigin>(
+				),
+				TypeId::of::<UnanimousCouncilVote>(),
+			);
+
+			assert_eq!(
+				TypeId::of::<UnanimousCouncilVote>(),
+				TypeId::of::<
+					EitherOfDiverse<
+						EnsureRoot<AccountId>,
+						pallet_collective::EnsureProportionAtLeast<
+							AccountId,
+							CouncilCollective,
+							1,
+							1,
+						>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn number_of_members_is_limited() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::MaxMembers>(
+				),
+				TypeId::of::<CouncilMaxMembers>(),
+			);
+
+			assert_eq!(CouncilMaxMembers::get(), 100);
+		}
+
+		#[test]
+		fn council_handles_membership_changes() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_membership::Config<CouncilMembership>>::MembershipChanged,
+				>(),
+				TypeId::of::<Council>(),
+			);
+		}
+
+		#[test]
+		fn council_handles_membership_initialization() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::MembershipInitialized>(),
+				TypeId::of::<Council>(),
+			);
+		}
+
+		#[test]
+		fn prime_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::PrimeOrigin>(
+				),
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+			);
+			assert_eq!(
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+				TypeId::of::<
+					EitherOfDiverse<
+						EnsureRoot<AccountId>,
+						pallet_collective::EnsureProportionAtLeast<
+							AccountId,
+							CouncilCollective,
+							3,
+							4,
+						>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn remove_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::RemoveOrigin>(
+				),
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+			);
+		}
+
+		#[test]
+		fn reset_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::ResetOrigin>(
+				),
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+			);
+		}
+
+		#[test]
+		fn swap_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::SwapOrigin>(
+				),
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+			);
+		}
+
+		#[test]
+		fn default_weights_are_not_used() {
+			assert_ne!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::WeightInfo>(
+				),
+				TypeId::of::<()>(),
+			);
+		}
+	}
+
+	mod motion {
+		use super::*;
+
+		#[test]
+		fn simple_majority_always_fails() {
```

Comment:
```suggestion
		fn simple_majority_is_never_origin() {
```

Maybe this is a more accurate name?